build/dependencies.props

```diff
@@ -84,6 +85,7 @@
     <HealthCheckSystem>2.2.1</HealthCheckSystem>
     <HealthCheckNetwork>2.2.2</HealthCheckNetwork>
     <HealthCheckDynamoDb>2.2.0</HealthCheckDynamoDb>
+    <HealthCheckCosmosDb>2.2.0</HealthCheckCosmosDb>
```

Comment:
If this package use preview packages, set also as preview version, 2.2.0-preview!